### ISSUE TITLE
AI_A2A_DISPATCHER: Too many CAP?

### ISSUE THREAD

**Issue by FlightControl-Master** [FlightControl-Master/MOOSE#662]
There send to be a problem with CAP being spawned too many times. Need to check.

**Comment by shadowze** [comment]
I use 5 x CAP in my mission and 5 are active (takes a couple of minutes for all to be spawned)
using 1.5.7 DCS World

**Comment by FlightControl-Master** [comment]
Do you have one squadron doing CAP or more squadrons?

CAP, behaves strange in 2.1. It seems there are more CAP spawned than the parameter specifies or if there are two squadrons, then somehow it seems the amount of CAP requested is spread out over the squadrons.

**Comment by thebgpikester** [comment]
Reproduced! But this is intermittent and I believe a timing issue. Even when setting 
A2ADispatcher:SetSquadronCapInterval( "TTR", **1**, 20, 21, 1 ) the ! is not honoured occasionally. Initially it seems fine, but it starts when replacing a CAP, sometimes you get 3 or more CAP flights depart. One flight tends to go to CAP (sometimes) the remainder RTB instantly and sit in a queue.

This is 2.1 pre, NTTR, one squadron, one capzone, A2ADispatcher:SetSquadronOverhead( "TTR", 1 ) A2ADispatcher:SetSquadronGrouping( "TTR", 2 ) or grouping 1 even. Very basic setup.

**Comment by FlightControl-Master** [comment]
I fixed this issue this morning. Two issues were fixed:

The amount of CAP is now correctly counted per squadron. There was indeed a bug. I counted the total CAO over all squadrons.

The start and repeating times per squadron for spawning CAP planes are now correctly calculated.
The changes are populated in a branch locally on my pc with a lot of other changes.
I hope to publish this week to master.
